Formula/rdkit.rb

```diff
@@ -34,7 +34,12 @@ def install
     python_executable = Formula["python@3.9"].opt_bin/"python3"
     py3ver = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
     py3prefix = Formula["python@3.9"].opt_frameworks/"Python.framework/Versions/#{py3ver}"
+    on_linux do
+      py3prefix = Formula["python@3.9"].opt_prefix
+    end
     py3include = "#{py3prefix}/include/python#{py3ver}"
+    # Get homebrew numpy location
```

Comment:
Same here, the comment about numpy is not needed, you are just describing what the line does, so it does not bring any useful information. Sorry I did not notice it before.